src/MultiplayerMod/Multiplayer/Configuration/GameEventBindings.cs

```diff
@@ -76,7 +76,7 @@ ExecutionLevelManager levelManager
         // TODO: Cursor update may be ignored if MouseMoved isn't triggered after the rate period.
         // TODO: Will be changed later (probably with current / last sent positions check).
         InterfaceToolEvents.MouseMoved += position => throttle10Hz.Run<UpdateCursorPosition>(
-            () => client.Send(new UpdateCursorPosition(client.Player, position))
+            () => client.Send(new UpdateCursorPosition(new UpdateCursorPositionEvent(client.Player, position)))
```

Comment:
I would avoid doing any kind of logic in subscriptions like here (getting client.Player and creating event wrapper).
Perhaps even passing a client.Player is redundant and might be calculated automatically upon command recieve.

Reply:
I don't get what's concerning in making events or using some logic in subs, could you elaborate pls?🤔

Reply:
> I don't get what's concerning in making events or using some logic in subs, could you elaborate pls?🤔

Turned out it will be changed, but I still have this question.